conan/tools/microsoft/toolchain.py

```diff
@@ -14,10 +16,37 @@ class MSBuildToolchain(object):
 
     filename = "conantoolchain.props"
 
+    _config_toolchain_props = textwrap.dedent("""\
```

Comment:
I don't disagree with this refactor (making it a class attribute, instead of an internal method implementation detail), but it makes challenging to see what is changing in the template.